src/app/builders/BuildersPage.tsx

```diff
@@ -20,16 +20,16 @@ export const BuildersPage = () => {
       </PageTitleContainer>
 
       <div data-testid="info" className="flex flex-col w-full items-start gap-2">
```

Comment:
removing the background from each container means we could just define the background for this div right?
same for the backing page

Reply:
Yes, the other approach would be to have its own background and override it.